### DISCUSSION TITLE
Release 1.2.1

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#6754] [General]
## borgbackup 1.2.1 release

Short changelog summary (changes since borg 1.1):

https://www.borgbackup.org/releases/borg-1.2.html

Long changelog (including upgrade and compatibility notes):

https://github.com/borgbackup/borg/blob/1.2.1/docs/changes.rst#version-121-2022-06-06

### Installation

If you use pip to install this, use: pip install pkgconfig ; pip install "borgbackup==1.2.1"

For other installation methods and more details, please see: https://borgbackup.org/

### Feedback

Discussion: ...

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/1.2.1'>Release 1.2.1</a>.</em>

**Comment by micahredding** [comment]
Thank you! Is there a release schedule for the Homebrew packages?

**Reply by ThomasWaldmann** [reply]
@m3nu ^

**Reply by m3nu** [reply]
Opened PR for Homebrew: https://github.com/Homebrew/homebrew-core/pull/103191

**Reply by micahredding** [reply]
Opened an issue for homebrew-tap: 
https://github.com/borgbackup/homebrew-tap/issues/17

**Comment by muchcharles** [comment]
Import tar!  Thanks this is great, especially so when stuck on WSL1 without FUSE.

**Comment by David-Else** [comment]
Hello, currently I have borg 1.1x on my RHEL 8 home computer and borg 1.1x on my Ubuntu 20.04.4 LTS remote server, both packages are what is supplied by default.

I am updating to RHEL 9 which has borg 1.2x, so I need to update my Ubuntu that does not have a 1.2x package.

- Would you advise I uninstall 1.1 from Ubuntu and instead use on the the prebuilt binaries? If so, which one? There is linux, linuxold and linuxnew... https://github.com/borgbackup/borg/releases/tag/1.2.1
- Would you advise I instead use pip to update borg on my Ubuntu... would it even work?

Maybe I should do it some other way, or maybe 1.1 and 1.2 are compatible?

Thanks.

**Reply by m3nu** [reply]
They are compatible. We switched thousands of repos to 1.2 and only saw a minor issue with index files that came up in very rare cases: https://github.com/borgbackup/borg/issues/6687

The upcoming Borg 2.0 will have breaking changes.

**Reply by viharm** [reply]
> * ... If so, which one? There is linux, linuxold and linuxnew... https://github.com/borgbackup/borg/releases/tag/1.2.1


Yes, which one should we use - `linux`, `linuxnew` or `linuxold`? what's the difference? I couldn't find anything in the docs.

PS: I love _borgbackup_ and grateful for your efforts in developing this awesome tool :-)

**Reply by ThomasWaldmann** [reply]
@viharm It's the build system used for the binaries and in the end, their glibc compatibility. Documented in the README, see where you got the binaries from.

If "linuxnew" works for you, use that one.
If it doesn't work (because you run some older linux), try "linux" and "linuxold".
